cmd/lastfm.go

```diff
@@ -60,11 +60,10 @@ func getAlbums(username string, period Period, count int, imageSize string) ([]A
 	// No albums to return
 	if len(lastFMResponse.TopAlbums.Album) == 0 {
 		log.Println("no albums!")
-		return nil, errors.New("No Albums found! Is the username correct?")
+		return nil, errors.New("no albums found! is the username correct?")
```

Comment:
`staticcheck` wants all errors to be lowercase. I kind of agree, but we'll need a better way to bubble the errors to the users.